frontend/app/src/components/home/Markdown.svelte

```diff
@@ -44,14 +46,23 @@
         });
     }
 
+    function replaceEveryone(text: string): string {
+        if (!text.includes("@everyone")) return text;
+        return text.replace(/(^|[\s(){}\[\]])@everyone($|[\s(){}\[\]])/g, (_match, _p1) => {
+            return " **@everyone** ";
```

Comment:
Is this meant to have the spaces on either side?

Reply:
yes

Reply:
Why? What if the message contains '(@everyone)', or starts /finished with @everyone?